manipulation_objects/README.md

```diff
@@ -2,16 +2,23 @@
 
 ## Overview
 
-This package contains USD assets for manipulation testing in Newton.
+This package contains USD assets used for manipulation in Newton.
 
 | Object | Folder | Description | Source |
 |--------|--------|-------------|--------|
 | Cup | `cup/` | A simple cup mesh | Custom modelled |
```

Comment:
Ok last nitpick - for source, the custom modelled is a bit unclear. Maybe say "Created by @lenroe-nv" or "Created by The Newton Developers" to make it clear.